Library/Homebrew/utils/popen.rb

```diff
@@ -13,7 +13,7 @@ def self.popen(args, mode)
         return pipe.read unless block_given?
         yield pipe
       else
-        $stderr.reopen("/dev/null", "w")
+        $stderr.reopen("/dev/null", "w") unless ARGV.debug? || ARGV.verbose?
```

Comment:
Can you make this an `if` instead; `unless` with multiple conditions are hard to parse. After that should be 🆗

Reply:
Done. What do you think of adding…
```ruby
$stderr.reopen("/dev/null", "w") if !ARGV.debug? && !ARGV.verbose? && !ENV["HOMEBREW_DEVELOPER"]?
```

Reply:
:-1: to that.

Reply:
Debug or verbose seems sufficient, thanks.

Reply:
That's fine.